orm/hibernate-orm-6/src/test/java/entities/InstantEntity.java

```diff
@@ -0,0 +1,32 @@
+package entities;
+
+import jakarta.persistence.*;
+
+import java.time.Instant;
+
+@NamedQuery(name = "InstantEntity.findBetween", query = "SELECT i from InstantEntity i where i.dateValue between :from and :to")
+@NamedQuery(name="InstantEntity.deleteAll", query="DELETE FROM InstantEntity i")
+@Entity
+@Table(name = "INSTANT_ENTITY")
+public class InstantEntity {
+    @Column(name = "DATE_VALUE", columnDefinition = "timestamp")
```

Comment:
Specifying a custom column definition is already asking for trouble. I think that the `TIMESTAMP_UTC` type expects `timestamp with time zone`, so if you want to use a different type, I think you'll have to annotate this with `@JdbcTypeCode(SqlTypes.TIMESTAMP)` or change the config like you described in your readme.